scripts/templates/webdriverio.tpl.d.ts

```diff
@@ -46,6 +46,13 @@ declare namespace WebdriverIO {
     type ServiceEntry = string | HookFunctions | [string, ServiceOption]
 
     interface Options {
+        /**
+         * Define the protocol you want to use for your browser automation.
+         * Currently only [`webdriver`](https://www.npmjs.com/package/webdriver) and
+         * [`devtools`](https://www.npmjs.com/package/devtools) are supported,
+         * as these are the main browser automation technologies available.
+         */
+        automationProtocol?: string;
```

Comment:
I am not sure if this can be modified in v6 anymore as it will be detected automatically. I have to check if this option is still recognised.

Reply:
i had issues in an early version of v6 and selenium standalone service and had to set automationProtocol to webdriver.  
i just checked it again with the current version and now it is detected automatically.  
let me know if i should remove this type definition

Reply:
just checked .. it is still being recognised: https://github.com/webdriverio/webdriverio/blob/master/packages/webdriverio/src/utils.js#L422-L427